src/Milestones/Block.php

```diff
@@ -53,6 +53,14 @@ public function addBlock() {
 						'type'    => 'string',
 						'default' => '',
 					],
+					'cta'         => [
```

Comment:
I would suggest being more explicit with the naming here. The Call to Action (text) and the URL are coupled attributes, but that isn't reflected here.

Reply:
I agree, it was getting pretty messy. Renamed attributes to linkText, linkUrl, and linkTarget. Ref: https://github.com/impress-org/givewp/pull/5262/commits/6ac4a31564bc33edbe1530d27060148415ba097c